core/src/main/java/org/apache/druid/java/util/http/client/response/FullResponseHolder.java

```diff
@@ -49,12 +49,7 @@ public HttpResponse getResponse()
   }
 
   /**
-   * Append a new chunk of data.
-   */
-  public abstract FullResponseHolder addChunk(T chunk);
```

Comment:
removed it because newly introduced `InputStreamFullResponseHolder.addChunk(byte[] chunk)` whereas `getContent()` needs to return `InputStream` . We could do the necessary wrapping so as to have `InputStreamFullResponseHolder.addChunk(InputStream chunk)` but would have required an extra wrap/unwrap of byte[] into/outof `ByteArrayInputStream` in `InputStreamResponseHandler .